docs/apache-airflow/howto/email-config.rst

```diff
@@ -29,7 +29,27 @@ in the ``[email]`` section.
   subject_template = /path/to/my_subject_template_file
   html_content_template = /path/to/my_html_content_template_file
 
-You can configure sender's email address by setting ``from_email`` in the ``[email]`` section.
+Equivalent environment variables look like:
+
+.. code-block::
+
+  AIRFLOW__EMAIL__EMAIL_BACKEND=airflow.utils.email.send_email_smtp
+  AIRFLOW__EMAIL__SUBJECT_TEMPLATE=/path/to/my_subject_template_file
+  AIRFLOW__EMAIL__HTML_CONTENT_TEMPLATE=/path/to/my_html_content_template_file
+
+You can configure a sender's email address by setting ``from_email`` in the ``[email]`` section like:
+
+.. code-block:: ini
+
+  [email]
+  from_email = "email@example.com"
+
+Equivalent environment variables look like:
+
+.. code-block::
+
+  AIRFLOW__EMAIL__FROM_EMAIL="From email <email@example.com>"
```

Comment:
```suggestion
  AIRFLOW__EMAIL__FROM_EMAIL="email@example.com"
```

Would show a valid email structure here too

Reply:
The config name is sort of a misnomer, the field is used by the email’s `From:` field, which takes a RFC 5321 address, not an email address. `From email <email@example.com>` is perfectly valid here.

Reply:
Wow, that's very unexpected, but thanks for clarifying. We could at least make it slightly less confusing by providing an example name, because showing `From email` does not tell the reader the expected value IMO. For example:

```
AIRFLOW__EMAIL__FROM_EMAIL=John Doe <"johndoe@example.com">
```